SimpleBlobDetector: No parameter 'minCircularity' is found in function set

On OpenCV 2.4 if I try to set (using Algorithm::set) minCircularity for SimpleBlobDetector I get an error:
```
OpenCV Error: Bad argument (No parameter 'minCircularity' is found) in set, file /builddir/build/BUILD/opencv-2.4.12.3/modules/core/src/algorithm.cpp, line 638
terminate called after throwing an instance of 'cv::Exception'
  what():  /builddir/build/BUILD/opencv-2.4.12.3/modules/core/src/algorithm.cpp:638: error: (-5) No parameter 'minCircularity' is found in function set
```
If I supply it as a parameter (SimpleBlobDetector::Params) and create the SimpleBlobDetector with its constructor it works, but Algorithm::set doesn't. To me, it seems that [modules/features2d/src/features2d_init.cpp](https://github.com/Itseez/opencv/blob/d4ec359f11bc71926d1308823095d9a9ebd95729/modules/features2d/src/features2d_init.cpp#L133) is missing some values that are defined in [modules/features2d/src/blobdetector.cpp](https://github.com/Itseez/opencv/blob/d4ec359f11bc71926d1308823095d9a9ebd95729/modules/features2d/src/blobdetector.cpp#L62). I don't think this is an issue with OpenCV 3.1 as it's different.

### Please state the information for your system
- OpenCV version: 2.4 
- Host OS: Linux (Fedora 23)

### In which part of the OpenCV library you got the issue?
features2d

### Expected behaviour
`simpleBlobDetector->set("minCircularity", 0.5f);`
Works. (0.5f is just an example value)

### Actual behaviour
It doesn't work, the error is at the beginning of this report.

### Partial code example
```
cv::Ptr<cv::FeatureDetector> glintDetector = cv::FeatureDetector::create("SimpleBlob");
glintDetector->set("minCircularity", 0.5f);
```

--- a/modules/features2d/src/features2d_init.cpp
+++ b/modules/features2d/src/features2d_init.cpp
@@ -139,12 +139,16 @@ CV_INIT_ALGORITHM(SimpleBlobDetector, "Feature2D.SimpleBlob",
                   obj.info()->addParam(obj, "filterByColor",    obj.params.filterByColor);
                   obj.info()->addParam(obj, "blobColor",        obj.params.blobColor);
                   obj.info()->addParam(obj, "filterByArea",     obj.params.filterByArea);
+                  obj.info()->addParam(obj, "minArea",          obj.params.minArea);
                   obj.info()->addParam(obj, "maxArea",          obj.params.maxArea);
                   obj.info()->addParam(obj, "filterByCircularity", obj.params.filterByCircularity);
+                  obj.info()->addParam(obj, "minCircularity",   obj.params.minCircularity);
                   obj.info()->addParam(obj, "maxCircularity",   obj.params.maxCircularity);
                   obj.info()->addParam(obj, "filterByInertia",  obj.params.filterByInertia);
+                  obj.info()->addParam(obj, "minInertiaRatio",  obj.params.minInertiaRatio);
                   obj.info()->addParam(obj, "maxInertiaRatio",  obj.params.maxInertiaRatio);
                   obj.info()->addParam(obj, "filterByConvexity", obj.params.filterByConvexity);
+                  obj.info()->addParam(obj, "minConvexity",     obj.params.minConvexity);
                   obj.info()->addParam(obj, "maxConvexity",     obj.params.maxConvexity);
                   )
 

